tests/Makefile

```diff
@@ -22,10 +30,26 @@ load-images: ## Load NGF and NGINX images on configured kind cluster
 	kind load docker-image $(PREFIX):$(TAG) $(NGINX_PREFIX):$(TAG)
 
 test: ## Run the system tests against your default k8s cluster
-	go test -v ./suite -args --gateway-api-version=$(GW_API_VERSION) --image-tag=$(TAG) \
+	go test -v ./suite $(GINKGO_FLAGS) -args --gateway-api-version=$(GW_API_VERSION) --image-tag=$(TAG) \
 		--ngf-image-repo=$(PREFIX) --nginx-image-repo=$(NGINX_PREFIX) --pull-policy=$(PULL_POLICY) \
-		--k8s-version=$(K8S_VERSION)
+		--k8s-version=$(K8S_VERSION) --service-type=$(GW_SERVICE_TYPE) --is-gke-internal-lb=$(GW_SVC_GKE_INTERNAL)
 
 .PHONY: delete-kind-cluster
 delete-kind-cluster: ## Delete kind cluster
 	kind delete cluster
+
+.PHONY: run-tests-on-vm
+run-tests-on-vm: ## Run the tests on a GCP VM
+	bash scripts/run-tests-gcp-vm.sh
+
+.PHONY: create-and-setup-vm
+create-and-setup-vm: ## Create and setup a GCP VM for tests
+	bash scripts/create-and-setup-gcp-vm.sh
+
+.PHONY: create-vm-and-run-tests
+create-vm-and-run-tests: create-and-setup-vm run-tests-on-vm ## Create and setup a GCP VM for tests and run the tests
+
+.PHONY: cleanup-vm
+cleanup-vm: ## Delete the test GCP VM and delete the firewall rule
```

Comment:
would it make sense to move this code to a script in scripts folder?
reasoning - this way, logic about names and stuff - all in one place - in the scripts folder